src/main/java/gmbs/domain/LottoTicket.java

```diff
@@ -0,0 +1,45 @@
+package gmbs.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+public class LottoTicket {
+    private final List<LottoNumbers> lottoTicket;
+
+    public LottoTicket(final List<LottoNumbers> lottoTicket) {
+        this.lottoTicket = lottoTicket;
+    }
+
+    public static LottoTicket createAutoLottoTicket(int count) {
+        return new LottoTicket(generateTickets(count));
+    }
+
+    private static List<LottoNumbers> generateTickets(int count) {
+        List<LottoNumbers> lottoTicket = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            lottoTicket.add(new LottoNumbers());
+        }
+        return lottoTicket;
```

Comment:
```suggestion
        return IntStream.range(0, count)
                .mapToObj(index -> new LottoNumbers())
                .collect(toUnmodifiableList());
```
IntStream으로 생성할 수도 있습니다~ 훨씬 간결해보이지 않나요?